docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Collect a distributed trace - .NET
+description: A tutorial to collect distributed traces in .NET applications
+ms.topic: tutorial
+ms.date: 03/14/2021
+---
+
+# Collect a distributed trace
+
+**This article applies to: ✔️** .NET Core 5.0 and later versions **or** any .NET application using the 
+[DiagnosticSource NuGet package](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/5.0.1) version 5 or later
+
+.NET applications can be instrumented using the <xref:System.Diagnostics.Activity?displayProperty=nameWithType> API to produce
+distributed tracing telemetry. In this tutorial you will record this instrumented telemetry with different telemetry collection
+libraries so that it is available to diagnose application issues. See 
+[the instrumentation tutorial](distributed-tracing-instrumentation-walkthroughs.md) if you need to add new instrumentation.
+
+## Collect using OpenTelemetry
+
+### Prerequisites
+
+- [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet) or a later version
+
+### Create an example application
+
+First you will create an example application that has some distributed trace instrumentation but nothing is being collected.
+
+```dotnetcli
+dotnet new console
+```
+
+Applications that target .NET 5 and later already have the necessary distributed tracing APIs included. For apps targeting older
+.NET versions add the [System.Diagnostics.DiagnosticSource NuGet package](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/)
+version 5.0.1 or greater.
+```dotnetcli
+dotnet add package System.Diagnostics.DiagnosticSource --version 5.0.1
+```
+
+Replace the contents of the generated Program.cs with this example source:
+```C#
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Sample.DistributedTracing
+{
+    class Program
+    {
+        static ActivitySource s_source = new ActivitySource("Sample.DistributedTracing");
+
+        static async Task Main(string[] args)
+        {
+            await DoSomeWork();
+            Console.WriteLine("Example work done");
+        }
+
+        static async Task DoSomeWork()
+        {
+            using (Activity a = s_source.StartActivity("SomeWork"))
+            {
+                await StepOne();
+                await StepTwo();
+            }
+        }
+
+        static async Task StepOne()
+        {
+            using (Activity a = s_source.StartActivity("StepOne"))
+            {
+                await Task.Delay(500);
+            }
+        }
+
+        static async Task StepTwo()
+        {
+            using (Activity a = s_source.StartActivity("StepTwo"))
+            {
+                await Task.Delay(1000);
+            }
+        }
+    }
+}
+```
+
+Running the app does not log anything yet
+```dotnetcli
+> dotnet run
+Example work done
+```
+
+### Add logging using OpenTelemetry
```

Comment:
I'd avoid the work "logging" here as it could be confused with the "logging" pillar of Observability. Would prefer to consistently use `tracing` in this doc.

Reply:
I reworded as "Collect a trace using OpenTelemetry".

I am guessing that customers will relate better to 'trace' or 'tracing data' as nouns rather than 'trace/tracing' as a verb. I'll try to be consistent using the word 'collect' when I am drawing attention to the work of capturing data and transmitting it to a backend.